rest/adminapitest/admin_api_test.go

```diff
@@ -241,6 +241,39 @@ func TestLoggingKeys(t *testing.T) {
 	assert.Equal(t, map[string]interface{}{}, noLogKeys)
 }
 
+func TestServerlessChangesEndpointLimit(t *testing.T) {
+	base.RequireNumTestBuckets(t, 2)
+	base.SetUpTestLogging(t, base.LevelInfo, base.KeyReplicate, base.KeyHTTP, base.KeyHTTPResp, base.KeySync, base.KeySyncMsg)
+	rt := rest.NewRestTester(t, nil)
+	defer rt.Close()
+
+	resp := rt.SendAdminRequest(http.MethodPut, "/_config", `{"max_concurrent_connections" : 2}`)
```

Comment:
If you do a `PUT` I'd be cautious of dropping some of the original configuration here as we discussed briefly with @IsaacLambat that I'm not sure we support doing a partial configuration update.

Reply:
My fault, I was thinking this was db/_config.